.githooks/post-checkout

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env sh
 
+PREVIOUS_HEAD=$1
+NEW_HEAD=$2
```

Comment:
Are these 2 variables needed? If it is just to let developers be aware of the other params, would it be better to change them to comment instead?